ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py

```diff
@@ -43,3 +43,36 @@ def test_paddle_layer_norm(
         bias=offset[0],
         epsilon=eps,
     )
+
+
+# instance_norm
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.instance_norm",
+    values_tuple=_generate_data_layer_norm(
+        available_dtypes=helpers.get_dtypes("float"),
+    ),
+    eps=st.floats(min_value=0.01, max_value=0.1),
+)
+def test_paddle_instance_norm(
+    *,
+    values_tuple,
+    normalized_shape,
```

Comment:
Hello @Ambro19 , I apologize for the delay. It seems like the `instance_norm` tests are not running because of this line as this parameter does not correspond to anything and thus, causes the test to fail. Can you try removing this line and then making sure the test passes successfully? Thanks!

Reply:
Hello @saeedashrraf, that's all-good no worries. Oh ok, I see. I am going to go ahead and to do it. I would reach out if any help's needed. Thank you for pointing this out!

Reply:
Hi @saeedashrraf, I've fixed. But I can't test it due to my paddle installation issue. I'm troubleshooting it. Thanks